### DISCUSSION TITLE
RMaker_Switch Program not working with latest versions

### DISCUSSION THREAD

**Discussion by Naren-5** [espressif/arduino-esp32#10047] [Q&A]
Hello All, I recently faced an issue with RMaker_Switch example Program not working with latest version: 3.0.3.
And found that it is working fine with version:  2.0.17. Refer link:  https://forum.arduino.cc/t/problems-with-rainmaker/1279829
Hope it will be solved in upcoming releases _/|\_
Thank you,

**Comment by me-no-dev** [comment]
fix on the way. the fix is to actually use larger partition scheme and also fix the maximum size of the current one, which allows larger firmware to pass than the ESP can take. I suggest you try with a larger partition scheme, like Huge App or something with more than 2MB APP partition

cc @P-R-O-C-H-Y
